pandas/tests/indexing/test_datetime.py

```diff
@@ -313,3 +314,20 @@ def test_loc_setitem_with_existing_dst(self):
                                 columns=['value'],
                                 dtype=object)
         tm.assert_frame_equal(result, expected)
+
+    def test_getitem_with_datestring_with_UTC_offset(self):
```

Comment:
hmm, I think our partial timestamp indexing tests are all over the place, so put in pandas/tests/indexes/datetimes/test_partial_slicing.py (future PR to consolidate tests in test_datetime.py to there)

Reply:
Moved.